README.md

```diff
@@ -46,7 +46,7 @@ For full details on the system requirements for the drivers, see the [system req
 
 On the client machine:
 - PHP 7.0.x, 7.1.x, or 7.2.x (7.2.0 and up on Unix, 7.2.1 and up on Windows)
-- A Web server such as Internet Information Services (IIS) is required. Your Web server must be configured to run PHP
+- If using a Web server such as Internet Information Services (IIS) or Apache, it must be configured to run PHP
```

Comment:
This should come at the end of the list (after the ODBC driver requirement); since the ODBC driver is a genuine prerequisite, it should come before any 'if' configuration requirements.